### ISSUE TITLE
Manage Animations Crashing Decky

### ISSUE THREAD

**Issue by SadnessAndOreos** [TheLogicMaster/SDH-AnimationChanger#50]
As of 9/25, when I try to open Manage Animations, it loads the page, freezes, and crashes Decky. The animations seem to be working as expected, and I can change animations without an issue. I am running Animation Changer v1.3.0-1 and Decky v2.10.5-pre2. This issue occurrs on the stable and prerelease versions of Decky.

**Comment by BeckysFootSlave** [comment]
Sadly I have the same problem!
Clicking on "manage animations" turns deck black and starts the security screen that I set up!
Without, it turns black and brings up the "homescreen"!

I think SteamOS 3.5 (in my case 3.6 I am on) broke animation changer in that point!)

Hope it can be fixed!

It's just "manage animations"

**Comment by eHerbaciarz** [comment]
I have the same problem on SteamOS 3.5 - option -"Force IPv4" doesn't resolve problem. Crash every time. On Steam OS 3.4.4 - everything was fine.

**Comment by TheLogicMaster** [comment]
It's an issue with the preview images. I have no idea what's causing the problem, though. Commenting out the src attribute of the RepoResultCard img and the video poster attribute of RepoResultModal prevents the issue, for some reason.

**Comment by Hakzource** [comment]
Yep, having the same exact issue. Although when I attempt to access the store twice in a row when it crashes and kicks me back to the deck lock screen, the entire decky plugin section disappears without a trace. It reappears when I reboot the entire system. Pressing "B"/Back also causes the deck to hang in a black screen until you force shutdown.
